lib/abstract_notifier/testing/minitest.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Minitest
+  module Assertions
+    def assert_delivery(args)
+      yield
+      assert_equal args, AbstractNotifier::Testing::Driver.deliveries.last
```

Comment:
We should better compare passed keyword arguments with the actual ones. Also, it makes sense to use an inclusion check here, so we can pass only some of the params to the assertion:

```ruby
assert_notifications_sent 2, title: event.title  do
  EventsNotifier.with(profile: profile).canceled(event).notify_now
  EventsNotifier.with(profile: another_profile).canceled(event).notify_now
end

assert_notifications_sent 1, identifier: "profile_id", title: event.title  do
  EventsNotifier.with(profile: profile).canceled(event).notify_now
 end
```